examples/moving_horizon_estimation/mhe.py

```diff
@@ -110,25 +112,26 @@ def get_solver_options(solver):
     mhe_dict = {"solver": None, "solver_options": None, "solver_options_first_iter": None}
     if solver == Solver.ACADOS:
         mhe_dict["solver"] = Solver.ACADOS
-        mhe_dict["solver_options"] = {
-            "nlp_solver_max_iter": 1000,
-            "integrator_type": "ERK",
-            "print_level": 0,
-        }
+        options = SolverOptionsAcados()
+        options.set_maximum_iterations(1000)
+        options.print_level = 0
```

Comment:
Do you think it makes to add a set_print_level since they are in both?

Reply:
Done.